app/src/main/res/layout/topic_summary_view.xml

```diff
@@ -56,13 +54,13 @@
       android:id="@+id/lesson_count_text_view"
```

Comment:
This view has a opacity of `80%` in mocks, check that.

Reply:
done.